src/views/Run/internal/ResultDisplay/internal/ReportSection/internal/TestResult/TestResult.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react'
+
+import { TestPredicate } from '@/benchttp/tests'
+import { Accordion } from '@/components'
+
+import { TestResultContent, TestResultTitle } from './internal/components'
+
+export type { IProps as ITestResultProps }
+
+export interface IProps {
```

Comment:
This syntax is weird, if we need to expose it correctly named, we should probably expose it with the name we want.